packages/injected/src/types.ts

```diff
@@ -172,7 +172,7 @@ export enum InjectedNameSpace {
   Arbitrum = 'arbitrum',
   XFI = 'xfi',
   GameStop = 'gamestop',
-  BitKeep = 'bitkeep',
+  Bitget = 'bitkeep',
```

Comment:
does `'bitkeep'` need to be updated?

Reply:
@Adamj1232 no, the `'bitkeep'` is the property key of `windows`, which is not changed.